README.md

```diff
@@ -36,12 +37,35 @@ Final render:
       <h1>Hello world</h1>
     </div>
 
+## React Native
+
 To activate React Native support you must pass in a `native` plugin option like so:
 
     plugins: [
       ["@fullstory/babel-plugin-annotate-react", { native: true }]
     ]
 
+### React Native with FullStory 
+
+When using this library with [FullStory for Mobile Apps](https://www.fullstory.com/platform/mobile-apps/), we recommend setting the `componentAttribute` and `elementAttribute` options both to `'fsTagName'` to generate better privacy selectors. 
```

Comment:
nit: add link to `fsTagName`

```suggestion
When using this library with [FullStory for Mobile Apps](https://www.fullstory.com/platform/mobile-apps/), we recommend setting the `componentAttribute` and `elementAttribute` options both to [`'fsTagName'`](https://help.fullstory.com/hc/en-us/articles/360052419133-Getting-Started-with-FullStory-React-Native-Capture#01GRMBR5FK3R89ZQQ9F3V6Z5RB) to generate better privacy selectors. 
```

Reply:
Yeah, that's a good idea.

Also, https://help.fullstory.com/hc/en-us/articles/360052419133-Getting-Started-with-FullStory-React-Native-Capture#01GRMBR5FK3R89ZQQ9F3V6Z5RB and https://developer.fullstory.com/mobile/react-native/auto-capture/set-tag-name/ should really link to eachother.